### PR TITLE
fix replying to audio files in bot pm.

### PR DESCRIPTION
**Pull request by subinps**

### CODE REVIEW

File: userge/plugins/utils/botpm.py [hunk 1]
```diff
@@ -446,9 +435,11 @@ async def handle_incoming_message(msg: PyroMessage):
             _USERS.append(user_id)
             await USERS.insert_one({"user_id": user_id})
         try:
-            if msg.sticker:
-                await bot.send_message(userge_id, f"{msg.from_user.mention} sent you a sticker.")
             m = await msg.forward(userge_id)
+            if msg.audio:
```
Comment (Krishna-Singhal) [inline userge/plugins/utils/botpm.py:439]: Don't Audio msg have forward tag ? also why did you remove `if msg.sticker:`

Comment (subinps) [reply userge/plugins/utils/botpm.py:439]: From September onwards official applications have forwarded tag for stickers.
And audio files don't have that yet.

Comment (Krishna-Singhal) [reply userge/plugins/utils/botpm.py:439]: add `group=-1` in [this line](https://github.com/UsergeTeam/Userge/blob/2b20e02610535a9cc4809d8776f8feb98ffc41cb/userge/plugins/help.py#L105)

**Review by Krishna-Singhal — APPROVED** [review]

**Comment by rking32** [comment]
Thanks
